articles/most-useful-nodejs-packages/index.md

```diff
@@ -0,0 +1,424 @@
+-   You only need to remember the Lodash functions hence easier to code.
+-   Even new programmers can understand Lodash.
+
+#### Other Utility Libraries Include:
+-   [`Underscore.js`](https://underscorejs.org/) - Underscore.js provides useful functional programming helpers without having to extend any built-in objects.
+-   [`Futil`](https://github.com/smartprocure/futil-js) - Futil is designed to complement Lodash's functional utilities.
+
+Lodash remains one of the most downloaded Node.js modules in the NPM repository. Try it out in the back-end and use their CDN for the front-end.
+
+### 5. Moment.js
+[Moment.js](https://momentjs.com/) is a lightweight JavaScript development tool for date and time manipulation. It makes dates and times easy to format, parse, validate, and internationalize using a clean and concise API.
+
+Node.js date objects are not that bad but it requires a lot of coding for complex parsing, validation, and displaying date and time objects.
+
+To use Moment.js with Node.js, install the module using the command `npm install moment`. Then simply use `require()` in your application as in the following example:
+
+```js
+const moment = require('moment');
+const today = moment();
+console.log(today.format());
+```
+**OUTPUT: 2020-08-27T09:21:49+01:00**
+
+Moment.js can still run from the browser. It creates a global moment object, which is used to access date and time parsing and manipulation. Include `<script>` as in the example below.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset ="UTF-8"></meta>
+        <title> MomentJS Date Object </title>
+    </head>
+    <body>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.24.0/moment.min.js"></script>
+        <script>
+        // You have a 'moment' global here
+         const today = moment();
+         console.log(today.format());
+        </script>
+    </body>
+</html>
+```
+
+Moment.js is an awesome Node.js package. It simplifies date and time parsing, manipulation and validation. If you have a Node.js application that uses date and time objects, consider trying Moment.js. It will help you to solve any date and time problems in your application.
+
+Moment.js supports international languages for different time and date formats which is a good thing to have in your application if you are developing an application with multiple languages.
+
+#### Other Validator and Formatting Libraries Include:
+-   [`Validator.js`](https://github.com/validatorjs/validator.js) - Validator.js is a library for string validators and sanitizing.
+-   [`Express-validator`](https://www.npmjs.com/package/express-validator) - Express-validator is an Express middleware for Validator.js.
+
+### 6. Request
+[Request](https://github.com/request/request/issues/3142) is a HTTP client framework for making `http` calls. It supports HTTPS and follows redirects by default. [Request npm](https://www.npmjs.com/package/request)
```

Comment:
Request has been depreciated, https://github.com/request/request/issues/3142. You should highlighting an alternative like Axios instead.